atlp-rwanda/hackers-ec-be
# #187419065 Fix Product availability status type from string to boolean

#### What does this PR do?

#### Description of Task to be completed?

Changing Product availability status type from string to boolean

#### How should this be manually tested?
- clone repo
- switch to branch fixing-productAvailabilityStatus
- run npm i to install packages
- create .env file and have all environment variables as they appear in .env-Example file
- run npm dev
- have buyer and seller logged in
- create a product for a seller, see if by default is set to false and try to update availability status
- and as buyer try to a retrieve that product

#### What are the relevant pivotal tracker/Trello stories?

#187419065
